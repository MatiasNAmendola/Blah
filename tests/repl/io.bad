readLine('hello')
readLine('hello', 'whatever')
isEOF('what')
isEOF(1, 2, 3)
