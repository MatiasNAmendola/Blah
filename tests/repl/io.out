<eval> unexpected arguments, expected readLine()
<eval> unexpected arguments, expected readLine()
<eval> unexpected arguments, expected isEOF()
<eval> unexpected arguments, expected isEOF()
